src/Makefile

```diff
@@ -529,6 +532,8 @@ install: all
 	@ln -sf $(SERVER_NAME) $(INSTALL_BIN)/$(ENGINE_CHECK_RDB_NAME)
 	@ln -sf $(SERVER_NAME) $(INSTALL_BIN)/$(ENGINE_CHECK_AOF_NAME)
 	@ln -sf $(SERVER_NAME) $(INSTALL_BIN)/$(ENGINE_SENTINEL_NAME)
+	@(cd ../ && ln -sf $(CONFIG_FILE) $(OLD_CONFIG_FILE))
```

Comment:
We decided that we don't need a symlink for the config file. It is not installed anyway. It's just an example config.